docs/rules-and-regulations.md

```diff
@@ -17,204 +16,101 @@ The Organization shall not arbitrarily discriminate on the basis of race, nation
 
-Both the OP and Business Systems Point must have user accounts with maximum administrative access to each of the Business Systems. 
+All Business System accounts need to be established with a Operations role account.  No Business Systems should be held in a member's name.
 
-All relevant Organization members must be given appropriate access to each Business System as determined by the Business Systems Point or the OP.
+All relevant Organization members must be given appropriate access to each Business System as determined by Foundation Operations.
 
 ### Section 7.03. Business System Documentation.
 
-A list of all the currently used Business Systems must be documented in the Org Wiki which is maintained by the Business System Sub-Quorum. 
+A list of all the currently used Business Systems must be documented by the Foundation Operations team. 
 
 A list of all Business Systems used in the past must be listed on the Archived List of Business Systems. 
 
-A list of all Business Systems that are being considered for the Organization, regardless of the area of the Organization using them, must be listed on the Candidate Business Systems and maintained in the Org Wiki. 
+A list of all Business Systems that are being considered for the Organization, regardless of the area of the Organization using them, must be listed on the Candidate Business Systems. 
 
 ### Section 7.04. Business System Selection.
 
-The approval of a majority of the Partners and Contributors shall determine which Business Systems will be utilized by the Organization.  In the event that a majority of the Partner and the Contributor cannot decide on which Business System shall be utilized, the Business Systems Point shall create a set of recommended Business Systems, indicating which system is preferred. Additionally, the Business Systems Point may mandate that only a particular Business System be used in certain types of uses. 
-
-### Section 7.05. Free Business Systems.
-
-Any Business System which may be procured by the Organization free of charge which is requested by at least two members may be procured at the discretion of the Business Systems Point in compliance with section 7.02 of these Rules and Regulations. 
-
-### Section 7.06. Paid for Business Systems.
-
-Once a Business System has been approved by a majority of the Partner Members, or is recommended by the Business System Point, it must also be approved by the OP, Business Systems Point, and Finance Point prior to the purchase of such Business Systems. 
-
-Access to Paid Systems may be rationed when needed (in order to manage costs) according to an objective calculation approved by the consent of 75% of the OP, Business Systems Point, and Finance Point.
-
+The approval of Business Systems is done by the Foundation team and the currenly Partners. If it is a system requiring payment the approval proccess will include the Finance team. 
```

Comment:
Will make minor edits to correct typo.

Reply:
Did you make the edit?  If not, what did you find and I can change it.